test/demo-logger-using-pvc/logger.py

```diff
@@ -0,0 +1,44 @@
+#! /usr/bin/python3
+
+import logging
+import socket
+import time
+import sys
+
+LOG_DIR = "/var/log/demoLogger/"
+HOSTNAME = socket.gethostname()
+LOG_FILE = LOG_DIR + HOSTNAME + ".log"
```

Comment:
We can bifurcate log filename build up further

LOG_DIR = "/var/log/"
APP = "demo-logger"
APP_INSTANCE  = APP + '-' + (socket.gethostname()).replace('.','-')
LOG_FILE = LOG_DIR + APP + '/' + APP_INSTANCE + ".log"

Reply:
addressed 👍